lib/api/apiUtils/quotas/quotaUtils.js

```diff
@@ -215,8 +216,9 @@ function validateQuotas(request, bucket, account, apiNames, apiMethod, inflight,
     let accountQuotaExceeded = false;
     let quotaEvaluationDuration;
     const requestStartTime = process.hrtime.bigint();
-    const bucketQuota = bucket.getQuota();
-    const accountQuota = account?.quota || 0;
+    const bucketQuota = bucket.getQuota() || 0n;
```

Comment:
do we have some sort of "functional" test, where the bucketInfo is retrieved from Mongo (and 'parsed' to BigInt) ?
should we wrap this in BigInt() constructor for extra safety? (`BigInt(bucket.getQuota())`) ?

Reply:
We have this test at the Zenko level, currently. I can update to be safe indeed, but the new BucketInfo class already do it in constructor or `setQuota`?

Reply:
Actually yes we have the `tests/quota/awsNodeSdk.js` file, it uses a mock for scuba but it really create the buckets and uses the quota APIs to change the quotas, that are then evaluated.